src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function Button(props) {
```

Comment:
make this an arrow function and export it
`export const Button = (props) => {....`

We could also have this take a custom className, that way each time we use this we can pass a style!